runtime/auth/aws-signing-tests/common/src/aws/smithy/kotlin/runtime/auth/awssigning/tests/AwsChunkedTestBase.kt

```diff
@@ -55,10 +55,6 @@ fun AwsChunkedReaderFactory.create(
 abstract class AwsChunkedTestBase(
     val factory: AwsChunkedReaderFactory,
 ) : HasSigner {
-    val CHUNK_SIGNATURE_REGEX = Regex("chunk-signature=[a-zA-Z0-9]{64}") // alphanumeric, length of 64
-    val CHUNK_SIZE_REGEX = Regex("[0-9a-f]+;chunk-signature=") // hexadecimal, any length, immediately followed by the chunk signature
-    val UNSIGNED_CHUNK_SIZE_REGEX = Regex("[0-9a-f]+\r\n")
```

Comment:
question: could we have kept these values?

Reply:
Yes we could have, but since they were only used once, I didn't really see the point of keeping them. I do think it helps readability so let me replace them. They just need to be made top-level `private val` instead of class members, because they are using SCREAMING_SNAKE_CASE

Reply:
Screaming snake case😆 thank you